opencti-platform/opencti-graphql/src/modules/indicator/indicator.ts

```diff
@@ -25,7 +25,7 @@ const INDICATOR_DEFINITION: ModuleDefinition<StoreEntityIndicator, StixIndicator
     { name: 'description', label: 'Description', type: 'string', format: 'text', mandatoryType: 'customizable', editDefault: true, multiple: false, upsert: true, isFilterable: true },
     { name: 'pattern_type', label: 'Pattern type', type: 'string', format: 'vocabulary', vocabularyCategory: 'pattern_type_ov', mandatoryType: 'external', editDefault: true, multiple: false, upsert: false, isFilterable: true },
```

Comment:
pattern_type & pattern_version don't need to be upsertable ?

Reply:
that is a good question. If we change the pattern we need to be able to change the type and version, otherwise you would change from stix to yara and keeping type as stix.... it's completly broken then.

I'll change the upsert too for these fields